.travis.yml

```diff
@@ -35,8 +35,12 @@ jobs:
       python: 3.6
       if: branch = master AND type = push
 
-    # OSX build removed due to long build startup delays
-    # Restore matrix job entry from d2d67fab to test OSX
+    - os: osx
+      sudo: true
+      python: 3.6
+      language: bash
```

Comment:
Why is it set to bash?

Reply:
https://travis-ci.org/Hori75/coala/jobs/396015861

I have tried without setting it to bash and I got command not found errors

Reply:
`pip` and `python` need to point to the python3 which is being installed.

I think you need to add `/usr/local/opt/python/libexec/bin` to the PATH
c.f.
https://travis-ci.org/Hori75/coala/jobs/396904486#L1411

then these lines shouldnt be necessary
```
sudo ln -sf $(which python3) $(which python)
sudo ln -sf $(which pip3) $(which pip 2>/dev/null || echo /usr/bin/pip)
```